docs/features/templates.md

```diff
@@ -85,9 +85,11 @@ $app = new \Slim\App();
 $container = $app->getContainer();
 
 // Register Twig View service
-$container->register(new \Slim\Views\Twig('path/to/templates', [
-    'cache' => 'path/to/cache'
-]));
+$container['view'] = function () {
+    return new \Slim\Views\Twig('path/to/templates', [
+        'cache' => 'path/to/cache'
+    ]);
+};
```

Comment:
I just followed the existing doc, which didn't mention TwigExtension in it. Should I still change this?